t04_refactored.py

```diff
@@ -21,7 +21,7 @@
 import random
 from time import sleep
 
-delay = 1.0          # change to 0.0 for testing/speed runs; larger for dramatic effect!
```

Comment:
Changes should not have been made outside of your function. Please revert these.